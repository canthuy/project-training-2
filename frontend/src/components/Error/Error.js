import { ErrorSC } from './Error.styles';

const Error = (props) => {
  return <ErrorSC>{props.message}</ErrorSC>;
};

export default Error;
