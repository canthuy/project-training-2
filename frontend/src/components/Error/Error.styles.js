import styled from 'styled-components';

const ErrorSC = styled.h2`
  text-align: center;
  color: #be0f0f;
  margin: 20px;
`;
export { ErrorSC };
