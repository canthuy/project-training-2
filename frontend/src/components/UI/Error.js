import styled from 'styled-components';

const Error = styled.h2`
  text-align: center;
  color: #be0f0f;
  margin: 20px;
`;
export default Error;
