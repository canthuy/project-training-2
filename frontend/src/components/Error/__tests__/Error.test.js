import React from 'react';
import { render } from '@testing-library/react';
import Error from '../Error';
window.React = React;

describe('Error Component', () => {
  it('render component', () => {
    const { container, getByText } = render(<Error message="error" />);
    expect(container).toMatchSnapshot();
    expect(getByText(/error/i)).toBeTruthy();
  });
});
