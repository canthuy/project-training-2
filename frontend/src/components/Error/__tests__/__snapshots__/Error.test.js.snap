// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Component render component 1`] = `
<div>
  <h2
    class="sc-bdvvtL cuTfRT"
  >
    error
  </h2>
</div>
`;
